src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -363,10 +385,13 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Authentication listeners
         list($authListeners, $defaultEntryPoint) = $this->createAuthenticationListeners($container, $id, $firewall, $authenticationProviders, $defaultProvider, $configuredEntryPoint);
 
+        $config->replaceArgument(6, $configuredEntryPoint ?: $defaultEntryPoint);
```

Comment:
As it is a native listener, I'll be in favor of adding a `isAnonymous` getter indeed.

> Could you give me your opinion about the FirewallConfig::$listenerConfigs property?

IMHO, as we discussed, as this is the raw configuration for listeners here, we should probably not expose it but instead simply list enabled listener for the given firewall.